server/config/questions.json

```diff
@@ -1,6 +1,43 @@
 [
     {
         "name": "Participant",
+        "text": [
```

Comment:
Why can't these `text` objects also be in the `questions` section? It seems much easier for the user to have the order of the JSON array represent the order of the form layout. Maybe just give them a different type like `label`?